### DISCUSSION TITLE
Cant get github student pack

### DISCUSSION THREAD

**Discussion by Gangrels** [community/community#55554] [GitHub Education]
### Hi everyone,

Hi,
I tried to go through the verification process several times already, but every time the verification system sends me the same answer:

>You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.

    I filled it out according to my name and email

>The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation.

    I have provided all the documents that confirm that I am indeed a student. I've tried taking pictures with my phone for better quality, I've tried taking pictures with the built-in webcam, but the answer is always the same. My name, the name of the educational institution, all the seal was clearly visible. My documents are not in English, and I don't understand how they need to be translated, I need to print the translation on paper or google lens translation will do?

>You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

    Since the war started in Ukraine and rocket hit our main campus, we switched to online studying. What document should i show, a photo of the destroyed university building?

Thanks

**Comment by JasonStephen** [comment]
Here is my advice for the first situation combined with my own experience:
Make sure you have them on your Profile and that they are correct.
1. You need to add "Student of + the name of your school "to your personal profile. Your school name should be the same as the name on your ID
2. The company should write your name, and your school name should be the same as the name on your ID, in English
3. Your name should also be in English, not in your local language (it matches your English credentials)
4. On the link, you can add the school link (at least better and safer than not).

This is my advice for the second situation combined with my own experience:
1. I have tried to translate it into English and print it out for shooting, but it seems that it can't be passed, but you can try.
2. Yes, it looks like the ID can only be in English, but you can also consider including your non-translated ID as well. That's how I passed. See Method 3 for details.
3. You can try to take multiple photos at the same time that can be used to prove your school status. I once tried to put only the mother tongue version and English version of the verification list of the network authority certification platform, only put the student ID card, only put the admission notice and so on, but when I put them all on and then take photos, I passed.

This is a suggestion for the third case
1. This problem is relatively rare, but if you can connect to your school's Intranet VPN, you can try it (not guaranteed, but it is a way to change the IP address so that the platform thinks you are at your school).
2. If no VPN is available or cannot be connected to the Intranet of the school, add remote study to the end of date of enrollment in the translation.

It took me one afternoon and one night to pass the exam after nearly 40 attempts. I hope my trial and error experience can help you. If you have more questions, you are welcome to discuss with me. I wish you success in your application.
Note: The reply content is translated through the translation software, if you don't understand, you can directly reply to me, I am online every day

**Reply by Gangrels** [reply]
Thank you very much for such a detailed answer. First of all, I will try to translate all the documents into English and print them out. Let's see how it helps to pass verification.

I'll answer as soon as I get a response from them
